pyuvdata/utils.py

```diff
@@ -34,6 +34,27 @@ def _str_to_bytes(s):
     def _bytes_to_str(b):
         return b.decode('utf8')
 
+# polarization constants
+# XXX is the 'p' capitalized or not?
+POLSTR_LIST = ['YX', 'XY', 'YY', 'XX', 'LR', 'RL', 'LL', 'RR', '', 'pI', 'pQ', 'pU', 'pV']
```

Comment:
I think we should use reversed dicts rather than these lists. Since the goal is to pull these out to make them useful to other codes, things like the need to add 8 to the value of the list location and the placeholder value in the `POLSTR_LIST` make it hard for other codes to use them correctly.